src/page-modules/assistant/__tests__/assistant.test.tsx

```diff
@@ -72,8 +72,11 @@ describe('assistant page', function () {
     (await expectProps(result)).toContain({
       initialFromFeature: fromFeature,
       initialToFeature: toFeature,
+      // initialSearchTime: {
+      //   mode: 'departBy',
+      //   dateTime: 123,
+      // },
```

Comment:
```suggestion
```

Reply:
Yes. The reason I commented this out is because the test fails if it's there, but I can't see why...